src/metax_api/tests/api/rest/base/views/common/auth.py

```diff
@@ -139,6 +148,46 @@ def test_end_user_create_access_error(self):
         response = self.client.post('/rest/files', {}, format='json')
         self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN, response.data)
 
+    @responses.activate
+    def test_additional_projects_success(self):
+        """
+        Ensures user's file projects are also fetched from local file.
+        """
+        self._use_http_authorization(method='bearer', token=get_test_oidc_token(new_proxy=True))
```

Comment:
I think it would be a good idea to put these "additional projects" test cases into their own group, by subclassing the `ApiEndUserAccessAuthorization` class, and name it something like `ApiEndUserAdditionalProjects`. That way you can also put the above line `self._use_http_authorization...` and `self._mock_token_validation_succeeds()` into its `setUp()`, instead of repeating everywhere.